gen/generate.go

```diff
@@ -326,24 +340,25 @@ func generateGenesisFiles(outDir string, protoVersion protocol.ConsensusVersion,
 		fmt.Fprintln(verboseOut, protoVersion, protoParams.MinBalance)
 	}
 
+	if rewardsBalance < protoParams.MinBalance {
+		// Needs to at least have min balance
+		rewardsBalance = protoParams.MinBalance
+	}
```

Comment:
I'm not sure about the "correct" behavior here if RewardsPoolBalance isn't set. Should the default value be used? I guess the answer is no, since this is changing the default to be 0.

Looks like it's no longer used, so you should remove  `defaultIncentivePoolBalanceAtInception`.

```suggestion
	if rewardsBalance < protoParams.MinBalance {
		// Needs to at least have min balance
		rewardsBalance = protoParams.MinBalance
	} else if rewardsBalance == 0
		rewardsBalance = defaultIncentivePoolBalanceAtInception
	}
```